lib/nerves_network/dhcp_manager.ex

```diff
@@ -44,6 +44,8 @@ defmodule Nerves.Network.DHCPManager do
     # was added to get things going.
     current_interfaces = Nerves.NetworkInterface.interfaces()
 
+    IO.puts "+++++++++++++++ current_interfaces = #{inspect current_interfaces}"
```

Comment:
might need to remove this debug statement.

Reply:
removed